src/coloring/acyclic_coloring.jl

```diff
@@ -0,0 +1,209 @@
+using LightGraphs
+using DataStructures
```

Comment:
Prefer the `using` statements in the central file of the package (`src/SparseDiffTools.jl`)

Also, since you are only using a limited number of things from DataStructures, add them explicitly, as in `using DataStructures: Type1, Type2, func`

Reply:
Done. Thanks!